src/main/java/org/zalando/pazuzu/feature/Feature.java

```diff
@@ -38,6 +41,17 @@
             inverseJoinColumns = @JoinColumn(name = "dependency_feature_id", nullable = false))
     public Set<Feature> dependencies;
 
+    public List<Tag> getTags() {
+        return tags;
+    }
+
+    public void setTags(List<Tag> tags) {
+        this.tags = tags;
+    }
+
+    @ManyToMany(fetch = FetchType.LAZY)
+    public List<Tag> tags;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/Pazuzu/pazuzu-registry/file/2764791626/issues/source?bid=3131965&fileBranchId=3355916#l53)